openwrt-openwisp-monitoring/files/monitoring.agent

```diff
@@ -196,6 +196,24 @@ send_data() {
 	done
 }
 
+wait_until_registered() {
+	if [ -n "$UUID" ] && [ -n "$KEY" ]; then
+		return 0
+	fi
+	logger -s "Waiting for device to register" \
+		-t openwisp-monitoring \
+		-p daemon.info
+	UUID=$(uci get openwisp.http.uuid 2> /dev/null)
+	KEY=$(uci get openwisp.http.key 2> /dev/null)
```

Comment:
I redirected standard error to `2> /dev/null` to avoid logging failures of the uci command.